cross/linux-arm-ssl-1.0.x.Dockerfile

```diff
@@ -8,16 +8,6 @@ RUN echo 'deb http://archive.debian.org/debian-security stretch/updates main' >>
 
 RUN apt-get update && apt-get -y install wget curl git make build-essential clang libz-dev libsqlite3-dev openssl libssl-dev pkg-config gzip mingw-w64 g++ zlib1g-dev libmpc-dev libmpfr-dev libgmp-dev gcc-aarch64-linux-gnu
 
-# cross compile OpenSSL
-# latest version can be found here: https://www.openssl.org/source/
-ENV OPENSSL_VERSION=openssl-1.0.2u
-ENV DOWNLOAD_SITE=https://www.openssl.org/source/old/1.0.2
-RUN wget $DOWNLOAD_SITE/$OPENSSL_VERSION.tar.gz && tar zxf $OPENSSL_VERSION.tar.gz
-RUN cd $OPENSSL_VERSION && ./Configure linux-aarch64 --cross-compile-prefix=/usr/bin/aarch64-linux-gnu- --prefix=/opt/openssl-arm --openssldir=/opt/openssl-arm -static && make install
```

Comment:
Not actually used here, we build openssl-1.0.x targets with `vendored-openssl` cargo feature so the vendored one takes precedence.